doc/workshop/facilitators.rst

```diff
@@ -30,6 +30,6 @@ Example: https://goo.gl/forms/UOkcsVROqV
 Vagrant boxes
 -------------
 
-See ``building.rst`` for instructions on building Vagrant boxes.
+See ``building.html`` for instructions on building Vagrant boxes.
```

Comment:
Shouldn't this be: :ref:`<building>` ? (and add .. _building to the building.rst page)

Reply:
thanks @flo-renaud, good catch. I missed this one. Fixed, rebased, and pushed.